cfgov/v1/models/molecules.py

```diff
@@ -117,16 +117,76 @@ class TextIntroduction(blocks.StructBlock):
 
     class Meta:
         icon = 'title'
-        template = 'v1/demo/molecules/text_introduction.html'
+        template = '_includes/molecules/text-introduction.html'
+
+
+class Hero(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(required=False)
+
+    image = ImageChooserBlock(required=False)
+    image_path = blocks.CharBlock(required=False)
+    image_alt = blocks.CharBlock(required=False)
+
+    background_color = blocks.CharBlock(max_length=100, required=False)
```

Comment:
I think that's actually a question for designers. I know there's at least a few colors they wanted to choose from, maybe we make it a dropdown? cc @ajbush @duelj